src/views/language/LanguageTable.tsx

```diff
@@ -112,6 +159,112 @@ const LanguageTable: React.FC = () => {
           sortParam: SortBy.PopulationOfDescendents,
           columnGroup: 'Population',
         },
+        {
+          key: 'Vitality (Meta)',
+          label: 'Vitality (Meta)',
+          render: (lang) => {
+            const { score, explanation } = computeVitalityMetascore(lang);
+            const b = scoreBucket(score);
+            return (
+              <Hoverable hoverContent={explanation}>
+                <div
+                  style={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    gap: '0.35em',
+                    color: bucketColor(b),
+                  }}
+                >
+                  {bucketIcon(b)}
+                  <span style={{ fontVariantNumeric: 'tabular-nums' }}>{scoreLabel(score)}</span>
+                </div>
+              </Hoverable>
+            );
+          },
+          isNumeric: true,
+          columnGroup: 'Vitality',
+        },
+        {
+          key: 'Vitality (ISO)',
+          label: 'Vitality (ISO)',
+          render: (lang) => {
+            const all = getAllVitalityScores(lang);
+            const s = all[VitalityMeterType.ISO].score;
+            const b = scoreBucket(s);
```

Comment:
Avoid single-letter variable names. I cannot express how much work I had to do in my first job when I inherited a machine-learning pipeline where everything was given names like "a, b, r, r2, rr"...

Also you never use the variable `all` a second time. Sometimes for code readability its good to have intermediate value, but in this case it does not seem necessary.

```suggestion
            const vitalityScore = getAllVitalityScores(lang)[VitalityMeterType.ISO].score;
            const scoreBucket = getScoreBucket(vitalityScore);
```